example-ideal.txt

```diff
@@ -0,0 +1,14 @@
+#start = corner
+N_sweep_move_W {
+  N_ -> N  // move north when there's not a wall
+  N! -> W S_sweep
```

Comment:
I like this idea of reducing Picobot's redundancy, but I was confused by the underscores and their different meanings. I initially thought that `N_` was a string like `N_sweep_move_W`.